src/engines/html5/media-source/adapters/native-adapter.js

```diff
@@ -136,30 +136,16 @@ export default class NativeAdapter extends BaseMediaSourceAdapter {
    * Checks if NativeAdapter can play a given drm data.
    * @function canPlayDrm
    * @param {Array<Object>} drmData - The drm data to check.
-   * @param {PKDrmConfigObject} drmConfig - The drm config.
    * @returns {boolean} - Whether the native adapter can play a specific drm data.
    * @static
    */
-  static canPlayDrm(drmData: Array<Object>, drmConfig: PKDrmConfigObject): boolean {
-    let canPlayDrm = false;
+  static canPlayDrm(drmData: Array<Object>): boolean {
     for (let drmProtocol of NativeAdapter._drmProtocols) {
-      if (drmProtocol.isConfigured(drmData, drmConfig)) {
+      if (drmProtocol.canPlayDrm(drmData)) {
         NativeAdapter._drmProtocol = drmProtocol;
-        canPlayDrm = true;
-        break;
       }
     }
-    if (!canPlayDrm) {
```

Comment:
see comment here: https://github.com/kaltura/playkit-js-dash/pull/82#discussion_r317394940
This is also valid here